resources/datahub/schema_aws_datahub.go

```diff
@@ -119,4 +131,10 @@ func (r *awsDatahubResource) Schema(_ context.Context, _ resource.SchemaRequest,
 			},
 		},
 	}
+	j, err := json.MarshalIndent(&resp, "", " ")
```

Comment:
Do we really need these debug logs? This is the schema response which is static from our code, no? I feel like this is excessive.

Reply:
yeah I can remove it, but ever since some of the values are going to be filled by automation I thought this can be a good idea for later debug if some of the params or operations are not logged properly (so we'll know what were the original input data for the execution)